crates/bevy_window/src/window.rs

```diff
@@ -624,6 +635,17 @@ impl Window {
     }
 }
 
+/// Defines where window should be placed at on creation.
+#[derive(Debug, Clone)]
+pub enum WindowPosition {
+    /// Position will be set by window manager
+    Default,
```

Comment:
```suggestion
    Automatic,
```

I think it's good to avoid confusing this with "the default value for this enum".

Reply:
I can get behind that.